src/Repository/Midata/QuestionnaireRepository.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Repository\Midata;
+
+use App\Entity\Midata\Group;
+use App\Entity\Midata\GroupType;
+use App\Entity\Quap\Questionnaire;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+
+class QuestionnaireRepository extends ServiceEntityRepository
+{
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, Questionnaire::class);
+    }
+
+    public function getQuestionnaireByGroup(Group $group): Questionnaire
+    {
+        if($group->getGroupType()->getGroupType() == GroupType::CANTON || $group->getGroupType()->getGroupType() == GroupType::REGION) {
```

Comment:
@SebastianStorz please use `in_array($group->getGroupType()->getGroupType(), [GroupType::CANTON, GroupType::REGION])` this makes it easier to read